reanalysis/utils.py

```diff
@@ -616,3 +616,14 @@ def good_string(in_string: str) -> bool:
     if isinstance(in_string, str) and len(in_string) > 0:
         return True
     return False
+
+
+def check_good_value(key: str, conf: dict[str, Any]) -> str | None:
+    """
+    check if a key from the dictionary is a good string
+    return it or None
+    """
+    conf_value = conf.get(key)
+    if good_string(conf_value):
+        return conf_value
+    return None
```

Comment:
What is the purpose of this function over conf.get(key)? Doesn't it just return None by default?
If you're planning on adding functionality here that makes sense then.

Reply:
There will be a template config file with this repo, with mostly blank values for the deployment-specific details. Those defaults should match the intended types, so the default value would be `""` instead of `none`. 

My thinking here is to `.get()` the value for a specific key, and return None either if the value is truly None, or if the value is an empty String. Definitely a cleaner way to do this...

Reply:
Ah nice. Yeah sounds good.